fl4health/losses/deep_mmd_loss.py

```diff
@@ -0,0 +1,224 @@
+from typing import Optional, Tuple
+
+import numpy as np
+import torch
+
+
+class ModelLatentF(torch.nn.Module):
+    """Latent space for both domains."""
+
+    def __init__(self, x_in: int, H: int, x_out: int):
```

Comment:
I'm not sure if `H` here has a specific meaning elsewhere, but it essentially looks like the hidden dimension of this kernel network. So maybe we can just call it `hidden_dimension`?